subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/tasks/JacocoReportBase.java

```diff
@@ -48,6 +51,7 @@
 @Incubating
 public abstract class JacocoReportBase extends JacocoBase {
 
+    private Set<JacocoTaskExtension> taskExtensions = new HashSet<JacocoTaskExtension>();
```

Comment:
I avoided that on purpose, because they can change after the task is added. 
I'm actually having second thoughts about the whole idea of transferring them from extensions. 
In jacoco includes/excludes are configured separately for the agent and for the report tasks, perhaps there is a reason for it.